### PR TITLE
Add Prometheus ETCD certs

### PR DESCRIPTION
**Pull request by wtty-fool**
Towards giantswarm/giantswarm#13333

## Checklist
- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/project/project.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "9.1.1"
+	version            = "9.1.1-jakub"
```
Comment (xh3b4sd) [inline pkg/project/project.go:8]: This should actually have the `-dev` prefix. @giantswarm/team-firecracker-engineers any ideas where this got lost recently? I thought the release workflows take care of that.

File: go.mod [hunk 2]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/giantswarm/apiextensions/v2 v2.5.1
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/certs/v3 v3.0.0
+	github.com/giantswarm/certs/v3 v3.0.1-0.20200925114212-d08b0085d6b0
```
Comment (xh3b4sd) [inline go.mod:11]: Let's tag that before merge. ❤️

**Review by xh3b4sd — APPROVED** [review]

**Comment by calvix** [comment]
Can you explain a little bit more? this is confusing as the ticket says that you need to access g8s Prometheus (which is on CP) and you are putting some certificates on TC?

File: pkg/project/project.go (hunk 1 above)
Comment (wtty-fool) [reply pkg/project/project.go:8]: I'll fix it manually for this PR. Actions have created a PR here: https://github.com/giantswarm/aws-operator/pull/2757, but it was never merged.

File: go.mod (hunk 2 above)
Comment (wtty-fool) [reply go.mod:11]: Sure thing, I'll update as soon as I release merged certs.

**Comment by wtty-fool** [comment]
@calvix 
> Can you explain a little bit more? this is confusing as the ticket says that you need to access g8s Prometheus (which is on CP) and you are putting some certificates on TC?

Good question. So what Biscuit wants is to convert TC into a CP of its own, with everything we run on a normal CP. This includes `g8s-prometheus`. Hence we deploy `g8s-prometheus` on TC.

> Why is a TC accessing a CP? the certs you are pushing here are probabbly certs for TC and not for CP to that might not work

`g8s-prometheus` deployed on a TC needs the certs to scrape TC's etcd. No CP certs needed :slightly_smiling_face:

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added etcd client certificates for Prometheus.
```
Comment (kopiczko) [inline CHANGELOG.md:13]:
```suggestion
- Add etcd client certificates for Prometheus.
```

**Review by kopiczko — APPROVED** [review]

**Comment by calvix** [comment]
> @calvix
> 
> > Can you explain a little bit more? this is confusing as the ticket says that you need to access g8s Prometheus (which is on CP) and you are putting some certificates on TC?
> 
> Good question. So what Biscuit wants is to convert TC into a CP of its own, with everything we run on a normal CP. This includes `g8s-prometheus`. Hence we deploy `g8s-prometheus` on TC.
> 
> > Why is a TC accessing a CP? the certs you are pushing here are probabbly certs for TC and not for CP to that might not work
> 
> `g8s-prometheus` deployed on a TC needs the certs to scrape TC's etcd. No CP certs needed slightly_smiling_face

Ok that makes sense, the naming is a little confusing :p

**Review by calvix — APPROVED** [review]

**Pull request merged**